packages/rest/src/lib/RequestManager.ts

```diff
@@ -290,10 +290,14 @@ export class RequestManager extends EventEmitter {
 			this.createHandler(hash.value, routeId.majorParameter);
 
 		// Resolve the request into usable fetch/node-fetch options
-		const { url, fetchOptions } = this.resolveRequest(request);
+		const { url, fetchOptions, isAuthenticated } = this.resolveRequest(request);
 
 		// Queue the request
-		return handler.queueRequest(routeId, url, fetchOptions, { body: request.body, files: request.files });
+		return handler.queueRequest(routeId, url, fetchOptions, {
+			body: request.body,
+			files: request.files,
+			auth: isAuthenticated,
```

Comment:
We have direct access to the original request here.

```suggestion
			auth: request.auth !== false,
```